logs/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
```

Comment:
an empty .gitignore file would've done, with an ignore in the root .gitignore, with an entry along the lines of

```
logs/
```

I like the empty file in `./logs` with an entry in `.gitignore` better, because it is more explicit as to what things are being ignored.